### PR TITLE
[WebProfilerBundle] Add a "preview" tab in mailer profiler for HTML email

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       |
| License       | MIT
| Doc PR        | no need

I did not work with mail for a long time, and I just discover the new entry in the profile. This is very nice 👏🏼 
But I miss a preview rendering, so here we go

image

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig [hunk 1]
```diff
@@ -141,6 +141,18 @@
                                             </div>
                                             {% if message.htmlBody is defined %}
                                                 {# Email instance #}
+                                                <div class="tab">
+                                                    <h3 class="tab-title">HTML preview</h3>
+                                                    <div class="tab-content">
+                                                        <pre class="prewrap" style="max-height: 600px">
+                                                            <iframe
+                                                                src="data:text/html;base64;charset=utf-8,{{ message.htmlBody()|raw|base64_encode }}"
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: where is this`base64_encode` filter defined ? I don't find it either in Twig nor in Symfony.

Also `|raw` looks weird to me here, as I don't see why a `base64_encode` filter would be configured using `pre_escape`

Comment (lyrixx) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: Oups, indeed this filter lives in my application! I'll fix it ASAP

Comment (stof) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: In SwiftmailerBundle, this is solved by doing the base64 encoding in the collector.

Comment (lyrixx) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: Thanks, I fixed it

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig [hunk 2]
```diff
@@ -141,6 +141,18 @@
                                             </div>
                                             {% if message.htmlBody is defined %}
                                                 {# Email instance #}
+                                                <div class="tab">
+                                                    <h3 class="tab-title">HTML preview</h3>
+                                                    <div class="tab-content">
+                                                        <pre class="prewrap" style="max-height: 600px">
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig:147]: why using a `pre` tag as a wrapper of the iframe ?

Comment (lyrixx) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig:147]: To force the mail "area"
But I might be removed, I did not test it.

**Pull request merged**